static/bidder-info/sspBC.yaml

```diff
@@ -0,0 +1,12 @@
+maintainer:
+  email: "wojciech.bialy@grupawp.pl "
```

Comment:
We require a group email address instead of a specific person. Please refer to our new adapter coding guide: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html#respect-the-rules

Reply:
Changed email to group alias.